### PR TITLE
UI layout

### PR DESCRIPTION
**Pull request by tingamapuro04**
I have worked on the UI-layout branch where i have created the UI of the online calculator using class components. I have digits from zero to nine, operands and an equal sign just as a complete calculator.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
```
Comment (mikethreels) [inline src/App.js]: - [ ] the App component should be class-based according to the requirements

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Alphonce, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] in the public and src folder there are still some files which are created by the CRA which need to be removed.
Check the comments under the review.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tingamapuro04** [comment]
Hello, I am not very sure which files to delete.  I don't if they will
affect the app

**Comment by tingamapuro04** [comment]
I have removed text, images created by the CRA, I have changed the App function to a class component as per the requirements

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @tingamapuro04, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**